scripts/gitrelease.mjs

```diff
@@ -14,10 +14,24 @@ if (tagExists.stdout !== '') {
 }
 
 const releaseMessage = `chore(release): ${nextVersion} [skip ci]`;
+
+echo`Checkout out branch`;
+await $`git fetch`;
```

Comment:
we can probably speed things up by only fetching the branch we are interested in

Reply:
indeed!